### DISCUSSION TITLE
Current protocol version?

### DISCUSSION THREAD

**Discussion by kpobrien** [jankae/LibreVNA#261] [General]
Hi Jan, 

While writing code to communicate with the LibreVNA I noticed some typos in Device_protocol_v13.pdf. I'll submit a PR to fix them in the tex file and update the pdf. I wanted to first ask if v13 is the current protocol version or whether I should create and edit a v14, etc, document instead.

Thank you for the excellent documentation!

Regards,
Kevin

**Comment by jankae** [comment]
Hi Kevin,

the version number refers to the protocol itself, not the documentation. That means you can leave it at 13. See this file for the currently used protocol version: https://github.com/jankae/LibreVNA/blob/5f2788467bdc738b8ad5c9ad71d1939484e89dfd/Software/VNA_embedded/Application/Communication/Protocol.hpp#L13

Looking forward to you PR,
Jan

**Reply by kpobrien** [reply]
Thanks! Submitted. I'll make another PR if I find anything else.
